doc/source/whatsnew/v0.23.0.txt

```diff
@@ -453,6 +453,7 @@ Reshaping
 - Bug in :func:`Dataframe.pivot_table` which fails when the ``aggfunc`` arg is of type string.  The behavior is now consistent with other methods like ``agg`` and ``apply`` (:issue:`18713`)
 - Bug in :func:`DataFrame.merge` in which merging using ``Index`` objects as vectors raised an Exception (:issue:`19038`)
 - Bug in :func:`DataFrame.stack`, :func:`DataFrame.unstack`, :func:`Series.unstack` which were not returning subclasses (:issue:`15563`)
+- Bug in :func:`DataFrame.drop`, `ValueError` now raises when dropping an `Index` that has duplicates
```

Comment:
- <code>\`ValueError\`</code> --> <code>\`\`ValueError\`\`</code> (double backticks)
- <code>\`Index\`</code> --> <code>\`\`Index\`\`</code> (double backticks)
- Add the issue number: <code>(:issue:\`19186\`)</code>